rules/windows/process_creation/proc_creation_win_double_ext_parent.yml

```diff
@@ -0,0 +1,24 @@
+title: Suspicious Double File Extention in ParentCommandLine
+id: 5e6a80c8-2d45-4633-9ef4-fa2671a39c5c
+status: experimental
+description: Detect when in suspicious Double File Extension is use in the ParentCommandLine
+references:
+    - https://www.virustotal.com/gui/file/7872d8845a332dce517adae9c3389fde5313ff2fed38c2577f3b498da786db68
+author: frack113
+date: 2023/01/06
+tags:
+    - attack.defense_evasion
+    - attack.t1036.007 
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ParentCommandLine|contains: 
+            - '.doc.lnk'
+            - '.docx.lnk'
+            - '.pdf.lnk'
+    condition: selection
+falsepositives:
+    - Unknown
+level: high
```

Comment:
I think it's best to update 1cdd9a09-06c9-4769-99ff-626e2b3991b8 to not have a similar rule to maintain.

Also you can add the Symantec link to the reference section